src/components/Earn.module.css

```diff
@@ -46,3 +46,30 @@
   border-color: var(--bs-gray-800);
   opacity: 100%;
 }
+
+.offerContainer {
+  border: 1px solid var(--bs-gray-200);
+  border-radius: 0.3rem;
+  padding: 1.25rem;
+  margin-bottom: 1.5rem;
+}
+
+:root[data-theme='dark'] .offerContainer {
+  border-color: var(--bs-gray-700);
+}
+
+.offerContainer .offerTitle {
+  width: 100%;
+  font-size: 1.2rem;
+  color: var(--bs-body-color);
+}
+
+.offerContainer .offerLabel {
+  color: var(--bs-gray-600);
+  font-size: 0.8rem;
+}
+
+.offerContainer .offerContent {
+  font-size: 0.8rem;
+  word-break: break-all;
+}
```

Comment:
That's the same styles as the existing FB box right?

Reply:
Yes.